pkg/streamingpromql/benchmarks/benchmarks.go

```diff
@@ -170,6 +181,12 @@ func TestCases(metricSizes []int) []BenchCase {
 		{
 			Expr: "sum by (le)(h_X)",
 		},
+		{
+			Expr: "sum by (l)(nh_X)",
+		},
+		{
+			Expr: "sum by (le)(nh_X)",
```

Comment:
I'm not sure this benchmark makes sense - there's no `le` label on the `nh_...` series.

(same applies for `sum by (le)(rate(nh_X[1m]))` benchmark below)